src/commands/database/mysql/dump.tsx

```diff
@@ -61,60 +50,44 @@ export class Dump extends ExecRenderBaseCommand<
     const databaseId = await withMySQLId(this.apiClient, this.flags, this.args);
     const p = makeProcessRenderer(this.flags, "Dumping a MySQL database");
 
-    const connectionDetails = await getConnectionDetailsWithPassword(
+    const name = await withConnectionDetails(
```

Comment:
In most cases we would call a such a function `run...`, in this case `runWithConnectionDetails`. I can also live with the current name, I would leave the decision up to you.

Reply:
Fixed with c2b21c04ef61497d6105ef83190dc26d299b004b